spec/pluto/operation/rotate_spec.cr

```diff
@@ -0,0 +1,55 @@
+require "../../spec_helper"
+
+describe Pluto::Operation::Rotation do
+  describe "#rotation" do
+    it "checks padding and radius" do
+      image = ga_sample
+
+      expect_raises(Exception, "Can't padding image and limit rotation by radius") do
```

Comment:
Sorry, I was about to merge the PR, but noticed this. I think it still need to be "pad" here?

Reply:
Feel free to go ahead and merge the PR after fixing this.

Reply:
Ha! Good catch. Thwarted by find-and-replace.